src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -535,12 +535,12 @@ export class TableComponent implements AfterViewInit, OnInit, OnChanges, OnDestr
     // this method was triggered by ngOnChanges().
     if (this.fetchData.observers.length > 0) {
       this.loadingIndicator = true;
+      const loadingSubscription = this.fetchData.subscribe(() => {
+        this.loadingIndicator = false;
+        this.cdRef.detectChanges();
+      });
+      this._subscriptions.add(loadingSubscription);
```

Comment:
@ivoalmeida need your opinion here as well. its mostly done because the fetchData always gets subscribe and increases the observable count to +1, which means that the refresh button will always show up in every table regardless of if it needs to be refreshed or not.

Reply:
Is there a table where it should be disabled and it is not? If so, does your change work there? This subscription will only get destroyed when you don't have a table component on the screen. This is just to avoid memory leaks.

Reply:
Got it. Removing these lines fixes the problem. Thanks

Reply:
Got it. I can see it works now. Thanks